src/www/ui/page/AdminGroupUsers.php

```diff
@@ -118,7 +118,7 @@ protected function handle(Request $request)
       if ($row['group_user_member_pk']) {
         continue;
       }
-      $otherUsers[$row['user_pk']] = $row['user_name'];
+      $otherUsers[$row['user_pk']] = $row['user_desc'] != '' ? $row['user_desc']. ' ('. $row['user_name'] .')' : $row['user_name'];
```

Comment:
It will be better to check empty description using the PHP function instead as the value from DB can be `NULL` as well which will evaluate to TRUE.

```suggestion
      $otherUsers[$row['user_pk']] = empty($row['user_desc']) ? $row['user_name'] : $row['user_desc']. ' ('. $row['user_name'] .')';
```

Reply:
Chaning it, thanks

Reply:
Changing it, thanks

Reply:
I think isset will do the thing if I am applying empty it is clearing the variable value